ladybug_geometry/geometry2d/_1d.py

```diff
@@ -34,7 +34,7 @@ def from_dict(cls, data):
         Args:
             data: A python dictionary in the following format
 
-        .. code-block:: json
+        .. code-block:: python
 
             {
             "p": [10, 0],
```

Comment:
You can also change situations like this over to use tuples instead of lists. Basically any dictionary where an array is being used to represent a point or vector should preferably be a tuple.